modules/line_item/src/Controller/CommerceLineItemController.php

```diff
@@ -65,14 +65,14 @@ public function add(CommerceLineItemTypeInterface $commerce_line_item_type) {
   /**
    * The title_callback for the entity.commerce_line_item.add_form route.
    *
-   * @param \Drupal\commerce_line_item\CommerceLineItemTypeInterface $commerce_line_item_type
+   * @param \Drupal\commerce_line_item\CommerceLineItemTypeInterface $commerceLineItemType
    *   The current line item type.
    *
    * @return string
    *   The page title.
    */
-  public function addPageTitle(CommerceLineItemTypeInterface $commerce_line_item_type) {
-    return $this->t('Create @label', array('@label' => $commerce_line_item_type->label()));
+  public function addPageTitle(CommerceLineItemTypeInterface $commerceLineItemType) {
```

Comment:
Have you confirmed manually that this works? I seem to recall the $commerce_line_item_type variable being magical, that's why I couldn't kill the commerce_ prefix.